# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/NavigatorLogin.h
# Commit: d07def1365a4
# Full Hash: d07def1365a49dc8ecbf5b6c10fdfbd5ec750932
# Author: pstanciu <pstanciu@mozilla.com>
# Date: 2025-06-11 18:38:51
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing WPT failures @lfedcm-identity.create-store-collect.tentative.sub.https.html
#   
#   This reverts commit 03d4474e360f28be0fb075379f4e4535bd34b252.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4e25ddaadc86 -r d07def1365a4 dom/credentialmanagement/identity/NavigatorLogin.h
--- a/dom/credentialmanagement/identity/NavigatorLogin.h	Wed Jun 11 13:42:13 2025 +0000
+++ b/dom/credentialmanagement/identity/NavigatorLogin.h	Wed Jun 11 18:04:27 2025 +0300
@@ -22,8 +22,8 @@
   NS_INLINE_DECL_CYCLE_COLLECTING_NATIVE_REFCOUNTING(NavigatorLogin)
   NS_DECL_CYCLE_COLLECTION_NATIVE_WRAPPERCACHE_CLASS(NavigatorLogin)
 
-  explicit NavigatorLogin(nsPIDOMWindowInner* aGlobal);
-  nsPIDOMWindowInner* GetParentObject() const { return mOwner; }
+  explicit NavigatorLogin(nsIGlobalObject* aGlobal);
+  nsIGlobalObject* GetParentObject() const { return mOwner; }
   virtual JSObject* WrapObject(JSContext* aCx,
                                JS::Handle<JSObject*> aGivenProto) override;
 
@@ -41,7 +41,7 @@
   virtual ~NavigatorLogin();
 
  private:
-  nsCOMPtr<nsPIDOMWindowInner> mOwner;
+  nsCOMPtr<nsIGlobalObject> mOwner;
 };
 
 }  // namespace mozilla::dom